src/h5web/toolbar/controls/ToggleGroup.tsx

```diff
@@ -63,7 +63,7 @@ type Props = ToggleGroupProps & {
   children: ReactNode;
```

Comment:
Should this be changed ?

Reply:
I guess so! `ReactElement<BtnProps>[]` does the trick.